flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -796,15 +783,16 @@ public void testBackendUsesRegisteredKryoSerializerUsingGetOrCreate() throws Exc
      * <p>This test should not fail, because de- / serialization of the state should not be
      * performed with Kryo's default {@link com.esotericsoftware.kryo.serializers.FieldSerializer}.
      */
-    @Test
+    @TestTemplate
     public void testKryoRegisteringRestoreResilienceWithRegisteredType() throws Exception {
```

Comment:
```suggestion
    void testKryoRegisteringRestoreResilienceWithRegisteredType() throws Exception {
```
could be package private